eZ/Bundle/EzPublishMigrationBundle/Command/LegacyStorage/RegenerateUrlAliasesCommand.php

```diff
@@ -0,0 +1,1054 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
Removed in https://github.com/ezsystems/ezpublish-kernel/pull/1781/commits/46bdcf56925787942f620193cdd2b00d9c23ea2b